quickstarts/gigamon/gigamon-hawk/config.yml

```diff
@@ -1,35 +1,34 @@
+# This is automatically created by the system, this field should not be included by the author when creating a new quickstart.
 id: d6b36ecf-bf99-475c-938f-370153db8e70
-slug: gigamon-newrelic
-title: Gigamon
+
+slug: gigamon-hawk
```

Comment:
Giving a fair warning here, last time we changed this slug it broke links to a handful of blogs and other marketing posts which have been out for a year. We changed it back pretty quickly because of that.

Reply:
Oof -- let me change it back and only submit the changes I need for the new quickstart. Good catch!